scene/animation/animation_tree.cpp

```diff
@@ -442,6 +442,28 @@ double AnimationNode::blend_input_ex(int p_input, double p_time, bool p_seek, bo
 	return blend_input(p_input, info, p_filter, p_sync, p_test_only);
 }
 
+void AnimationNode::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	String pf = p_function;
+	if (p_idx == 0) {
+		if (pf == "find_input") {
+			for (AnimationNode::Input E : inputs) {
+				r_options->push_back(E.name.quote());
+			}
+		} else if (pf == "get_parameter" || pf == "set_parameter") {
```

Comment:
Some parameters are read_only, so they should not expose the setter.

Reply:
And how is them being "read only" determined?

Reply:
Look into `_is_parameter_read_only()`.